featurizers/fingerprints.py

```diff
@@ -261,3 +265,68 @@ def _calculate_fingerprint(
         }
 
         return np.array([GetErGFingerprint(x, **fp_args) for x in X])
+
+
+class MAP4Fingerprint(FingerprintTransformer):
+    def __init__(
+        self,
+        dimensions: int = 1024,
+        radius: int = 2,
+        is_counted: bool = False,
+        is_folded: bool = False,
+        random_state: int = 0,
+        n_jobs: int = None,
+        verbose: int = 0,
+    ):
+        super().__init__(n_jobs, verbose)
```

Comment:
I suggest passing by name, so that if we change the parent class, we won't have to change all inheriting classes